docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md

```diff
@@ -1,8 +1,215 @@
 # BigDL-Nano PyTorch Inference Overview
 
-add a link for examples here.
+BigDL-Nano provides serveral APIs which can easily apply optimizations on inference pipeline to improve latency and througput. Currently, performance accelerations are achieved by integrating extra runtime as inference backend engine or using quantization methods on full-precision trained models to reduce computation during inference. Trainer (`bigdl.nano.pytorch.Trainer`) provides the APIs for all optimizations you need for inference.
```

Comment:
Maybe "which can be easily used to apply optimizations"? Feels weird to say "APIs apply optimizations".

"on inference pipeline" -> "on inference pipelines" ?

"extra runtime as inference backend engine" -> "extra runtimes as inference backend engines" ?